paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -235,6 +247,13 @@ private fun PaymentSheetContent(
             }
         }
 
+        if (
+            currentScreen is PaymentSheetScreen.SelectSavedPaymentMethods &&
+            currentScreen.cvcRecollectionState is CvcRecollectionState.Required
+        ) {
+            CvcRecollectionField(currentScreen.cvcRecollectionState.cvcControllerFlow)
```

Comment:
this should be in the `Content` function of SelectSavedPaymentMethods instead of in `PaymentSheetContent` itself. Maybe even within `SavedPaymentMethodTabLayoutUI`? Not sure about that

Reply:
Good call! Updated 👍